ni/abc.py

```diff
@@ -101,10 +101,9 @@ class CLAHost(abc.ABC):
 
     @abc.abstractmethod
     async def check(self, client: aiohttp.ClientSession,
```

Comment:
This should get renamed since `not check()` doesn't clearly convey that  there's no CLA issues. Otherwise successful checks should be included in the dict.

Reply:
What name do you suggest?

1. `validate`
1. `find_problems`

Reply:
`problems`?